.github/workflows/greetings.yml

```diff
@@ -0,0 +1,14 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      continue-on-error: true
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: "Thank you for collaborating with the project by giving us feedback!"
+        pr-message: "Thank you for collaborating with the project to help us to improve! We hope the plugin is useful to you. Cheers!"
```

Comment:
I'm not sure if calling this a plugin is accurate, since node refers to it as just a package.
Also, changing `to help us to improve` to `to help us improve` may make the comment sound more natural

Reply:
@5uso-sng I have modified this with the word "CLI", as this repository is for a CLI. Please, check when you want.